curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/contour.md

```diff
@@ -61,10 +64,12 @@ tests:
 <h2>OpenCV.js</h2>
 <img id="imageSrc" src="https://s3.amazonaws.com/freecodecamp/FCCStickers-CamperBot200x200.jpg"/>
 <p></p>
+<canvas id="canvasCanny" ></canvas>
+<p></p>
 <canvas id="canvasOutput" ></canvas>
 
 <script async src="https://docs.opencv.org/master/opencv.js"
-        onload='cv["onRuntimeInitialized"]=()=> { contour() }'
+        onload='cv["onRuntimeInitialized"]=()=> { runSample() }'
```

Comment:
Maybe just `run`?

Reply:
IDN,  but i think <code>runSample()</code> is more informative.

Reply:
Ok, let's keep `runSample`